includes/media_bynder.utils.inc

```diff
@@ -27,27 +27,59 @@ function media_bynder_rest_client() {
  * @todo add support for type and effect
  */
 function media_bynder_embed_url($fragment_id, $type=false, $effects=false, $name = null){
-    $current_media = variable_get('current_media');
+    if (!$type) {
+        $type = "web";
+    }
+    $base_url = variable_get('media_bynder_url', '');
+    if (substr($base_url, -1) != '/') {
+        $base_url .= '/';
+    }
+
+    if (validateId($fragment_id) == FALSE) {
+        return FALSE;
+    }
 
-    if(isset($current_media)) {
-        if ($current_media['idHash'] == $fragment_id) {
-            return $current_media['thumbnails']['thul'];
+    $base_url .= 'm/' . $fragment_id . '/web-drupal.jpg';
+    $query = [];
+
+    if ($effects){
+        foreach($effects as $effect_key => $effect){
+            if ($effect_key == 'image_resize'){
+                $query['h'] = $effect['height'];
+                $query['w'] = $effect['width'];
+            }
         }
     }
-     return FALSE;
+    if (empty($query)) {
```

Comment:
I agree with @happylinks because it supports PHP 4 and besides: it just checks if the array is empty or not. I can change to code to:

if (count($query) == 0) if that would take away your concerns @geonetix